src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -2167,10 +2074,10 @@ namespace HVACMultiSpeedHeatPump {
         if (allocated(state.dataZoneEquip->ZoneEquipConfig) && MSHeatPump(MSHeatPumpNum).MyCheckFlag) {
             int zoneNum = MSHeatPump(MSHeatPumpNum).ControlZoneNum;
             int zoneInlet = MSHeatPump(MSHeatPumpNum).ZoneInletNode;
-            int coolingPriority = 0;
-            int heatingPriority = 0;
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:2170]:(style),[variableScope],The scope of the variable 'coolingPriority' can be reduced.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:2171]:(style),[variableScope],The scope of the variable 'heatingPriority' can be reduced.